app/controllers/api/v1/customers/transactions_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::V1::Customers::TransactionsController < Api::V1::BaseController
   def index
-    transactions = Customer.find(params[:id]).transactions
+    @transactions = Customer.find(params[:id]).transactions
 
-    respond_with transactions
+    respond_with @transactions
```

Comment:
@pwentz Since we're using JBuilders, I think you can remove all of these instances of `respond_with`.